pkg/controller/installation/core_controller.go

```diff
@@ -380,27 +406,6 @@ func updateInstallationWithDefaults(ctx context.Context, client client.Client, i
 		return err
 	}
 
-	var openshiftConfig *configv1.Network
```

Comment:
This logic was only used for defaulting IP pools, so it has moved to the IP pool controller.